Content.Server/Database/ServerDbBase.cs

```diff
@@ -359,6 +359,37 @@ public async Task AssignUserIdAsync(string name, NetUserId netUserId)
         public abstract Task AddServerRoleUnbanAsync(ServerRoleUnbanDef serverRoleUnban);
         #endregion
 
+        #region Role Timers
+        public async Task<RoleTimer> GetRoleTimer(Guid player, string role)
+        {
+            await using var db = await GetDb();
+
+            return await db.DbContext.RoleTimer
+                .Include(v => v.PlayerId == player)
+                .Include(v => v.Role == role)
```

Comment:
You want `.Where()` instead. `Include()` is for joining data in the query results.